_api-reference/index-apis/force-merge.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Force merge
+parent: Index APIs
+nav_order: 37
+---
+
+# Force merge
+**Introduced 1.0**
+{: .label .label-purple }
+
+The force merge API operation forces a merge on the shards of one or more indexes. For a data stream, the API forces a merge on the shards of the stream's backing index.
+
+## The merge operation
+
+In OpenSearch, a shard is a Lucene index, which consists of _segments_ (or segment files). Segments store the indexed data. Periodically, smaller segments are merged into larger ones and the larger segments become immutable. Merging reduces the overall number of segments on each shard and frees up disk space. 
+
+## Deleted documents
+
+When a document is deleted from an OpenSearch index, it is not deleted from the Lucene segment but is only marked to be deleted. When the segment files are merged, deleted documents are removed (or _expunged_). Thus, merging also frees up space occupied by documents marked as deleted.
+
+## Force Merge API
+
+In addition to periodic merging, you can force a segment merge using the Force Merge API. 
+
+Use the Force Merge API on an index only after all write requests to the index are completed. Force merge can produce very large segments. If write requests are still sent to the index, then the merge policy does not merge these segments until they primarily consist of deleted documents. This can increase disk space usage and lead to performance degradation.
```

Comment:
```suggestion
Use the Force Merge API on an index only after all write requests sent to the index are completed. Force merge can produce very large segments. If write requests are still sent to the index, then the merge policy does not merge these segments until they primarily consist of deleted documents. This can increase disk space usage and lead to performance degradation.
```